src/ssl.ml

```diff
@@ -261,91 +261,161 @@ external set_host : socket -> string -> unit = "ocaml_ssl_set1_host"
 
 external set_ip : socket -> string -> unit = "ocaml_ssl_set1_ip"
 
-external write : socket -> Bytes.t -> int -> int -> int = "ocaml_ssl_write"
-
-external write_substring : socket -> string -> int -> int -> int = "ocaml_ssl_write"
-
-external write_bigarray : socket -> bigarray -> int -> int -> int = "ocaml_ssl_write_bigarray"
-
-external write_bigarray_blocking :
-  socket -> bigarray -> int -> int -> int = "ocaml_ssl_write_bigarray_blocking"
-
-external read : socket -> Bytes.t -> int -> int -> int = "ocaml_ssl_read"
-
-external read_into_bigarray :
-  socket -> bigarray -> int -> int -> int = "ocaml_ssl_read_into_bigarray"
-
-external read_into_bigarray_blocking :
-  socket -> bigarray -> int -> int -> int = "ocaml_ssl_read_into_bigarray_blocking"
-
-external accept : socket -> unit = "ocaml_ssl_accept"
-
-external flush : socket -> unit = "ocaml_ssl_flush"
-
-external ssl_shutdown : socket -> bool = "ocaml_ssl_shutdown"
-
-let open_connection_with_context context sockaddr =
-  let domain = Unix.domain_of_sockaddr sockaddr in
-  let sock =
-    Unix.socket domain Unix.SOCK_STREAM 0 in
+module SslReleaseExt = struct
```

Comment:
what does `SslReleaseExt`, `SslComExt` etc mean?

I can probably guess `SslReleaseExt` means "not releasing the runtime lock", but what's `SslComExt`?

Could you add more descriptive names, even if they're longer?

Reply:
I addressed this by commit [cb3cc1f](https://github.com/savonet/ocaml-ssl/pull/106/commits/cb3cc1f8497fbafef4951a33b647e56b03f54c7b)